iset.mm

```diff
@@ -105599,6 +105599,31 @@ which defines the set we are taking the supremum of must (a) be true at
       TUMSEASEVBVDLUKUMTATUMAUBQUCUOVBURUQGUNBRNUDUEUFUGUHUJ $.
   $}
 
+  ${
+    $d x y A $.  $d x y B $.  $d x y C $.
+    $( Relationship between the order of an element and that of a multiple.  (a
+       divisibility equivalent).  (Contributed by Stefan O'Rear,
```

Comment:
Hard to understand...
Maybe:
> An integer divides another integer iff it divides its GCD with that other integer (stated with an additional factor `B` in the multiples). This can be used to compute orders of iterates of elements in groups: if `A` is the order of an element `x` in a group, then the order of `x^C`, say `D`, satisfies `A = ( D x. ( C gcd  A ) )`.

Reply:
> Hard to understand... Maybe:
> 
> > An integer divides another integer iff it divides its GCD with that other integer (stated with an additional factor `B` in the multiples). This can be used to compute orders of iterates of elements in groups: if `A` is the order of an element `x` in a group, then the order of `x^C`, say `D`, satisfies `A = ( D x. ( C gcd  A ) )`.

Feel free to make a follow up pull request for this. I'm afraid I don't understand this comment, or the set.mm one, or the one I put in iset.mm (a minor edit of the set.mm one), well enough to be an especially good reviewer, however.